doc/source/reference/random/index.rst

```diff
@@ -32,7 +32,7 @@ instance's methods are imported into the numpy.random namespace, see
 Quick Start
 -----------
 
-By default, `~Generator` uses normals provided by `~pcg64.PCG64` which will be
+By default, `~Generator` uses sequences provided by `~pcg64.PCG64` which will be
 statistically more reliable than the legacy methods in `~.RandomState`
```

Comment:
Maybe "bit sequences" instead? Also, I think "which has better statistical properties than the legacy mt19937 random number generator in ~.RandomState" would read better.

Reply:
:+1: to both. Or even just "bits" instead of "bit sequences".